debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/gc/GCArrayletObjectModelBase_V1.java

```diff
@@ -156,27 +146,20 @@ protected UDATA getSpineSizeWithoutHeader(long layout, UDATA numberArraylets, UD
 		UDATA spineArrayoidSize = new UDATA(0);
 		UDATA spinePaddingSize = new UDATA(0);
 
-		/* The spine consists of three (possibly empty) sections, not including the header:
+		/* The spine consists of three or four (possibly empty) sections, not including the header:
 		 * 1. the alignment word - padding between arrayoid and inline-data
 		 * 2. the arrayoid - an array of pointers to the leaves
 		 * 3. in-line data
-		 * In hybrid specs, the spine may also include padding for a secondary size field in empty arrays 
+		 * 4. A secondary size field, for hybrid arrays only.
```

Comment:
notion of hybrid is gone, more appropriate would be discontiguous.

Reply:
Good point, will update 👍

Reply:
well, I got somewhat confused. concept of hybrid array layout will continue to exist. while the concept of hybrid arraylet header is about to be removed.

so to be really correct, this (4th point) should say the secondary size field exists for any hybrid and discontiguous array layout (or simpoly any non-contiguous arraylayout), right?